BraveWallet/Crypto/Stores/Address.swift

```diff
@@ -26,7 +26,12 @@ extension String {
   var removingHexPrefix: String {
     hasPrefix("0x") ? String(dropFirst(2)) : self
   }
-  
+
+  /// Adds the `0x` prefix that if it does not exist on the string
+  public var addingHexPrefix: String {
+    hasPrefix("0x") ? self : "0x\(self)"
+  }
```

Comment:
do we need this to be used in target other than BraveWallet?

Reply:
I don't think it will be used in any other targets

Reply:
so we can remove the public?

Reply:
Ah yes thanks, I'll update this